model/host/host.go

```diff
@@ -2663,6 +2663,22 @@ func (h *Host) MarkShouldExpire(expireOnValue string) error {
 	)
 }
 
+// UnsetHomeVolume disassociates a home volume from a (stopped) host.
+// This is for internal use, and should only be used on hosts that
+// will be terminated imminently; otherwise, the host will fail to boot.
+func (h *Host) UnsetHomeVolume() error {
```

Comment:
It looks like the function is not currently being called anywhere. Do we plan to add a button on the UI or an api route? 

Whatever we end up doing with it, I think we should log it somewhere so we can trace it and keep track of it.

Reply:
It'll be used [EVG-17485](https://jira.mongodb.org/browse/EVG-17485) — that's a huge ticket so figured I'd take this opportunity to break it up a tiny bit. The job that unsets the home volume will definitely have logging if that seems sufficient to you!